knexfile.js

```diff
@@ -0,0 +1,15 @@
+var env = require('node-env-file');
+env(__dirname + '/.env');
+
+module.exports = {
+
+  development: {
+    client: 'mysql',
+    connection: {
+      database: process.env.dbName,
```

Comment:
All env variables should be capitalized. So here it would be `DBNAME`, for example.

Reply:
resolved.